src/pokeball.c

```diff
@@ -609,6 +616,19 @@ static void Task_DoPokeballSendOutAnim(u8 taskId)
     PlaySE(SE_BALL_THROW);
 }
 
+static void DoPokeballSendOutSoundEffect(void)
```

Comment:
```suggestion
static inline void DoPokeballSendOutSoundEffect(void)
```
prolly doesn't matter but would still make it an inline